devices.json

```diff
@@ -0,0 +1 @@
+{"devices": [{"ip": "192.168.1.1", "username": "admin", "password": "password"}, {"ip": "192.168.1.2"}, {"ip": "192.168.1.3"}]}
```

Comment:
I would much rather just have 1 device defined per line, and we load 1 line at a time into JSON.